internal/move2kubeapi/handlers/inputs.go

```diff
@@ -53,6 +54,16 @@ func HandleCreateProjectInput(w http.ResponseWriter, r *http.Request, isCommon b
 	}
 	r.Body = http.MaxBytesReader(w, r.Body, common.Config.MaxUploadSize)
 	if err := r.ParseMultipartForm(common.Config.MaxUploadSize); err != nil {
+		maxErr := &http.MaxBytesError{}
+		if ok := errors.As(err, &maxErr); ok {
+			logrus.Errorf("request body exceeded max upload size of '%d' bytes. Error: %q", common.Config.MaxUploadSize, err)
+			sendErrorJSON(
+				w,
+				"Request body exceeded max upload size. Try using a smaller input. Use the '--max-upload-size' flag while starting the server to increase the limit.",
```

Comment:
Shouldn't the method say the CR config too for those trying to control using CR?

Reply:
The CR config is the same flag